Source/Scene/Cesium3DTileStyle.js

```diff
@@ -246,13 +246,24 @@ define([
                 return this._show;
             },
             set : function(value) {
+                if (typeof value === 'boolean') {
+                    this._show = new Expression(String(value));
```

Comment:
I think this should be a separate PR, because we don't have a getter/setter for defines now. So I don't know if you really want old `defines` are still evaluated. And in case you allow getter and setter for `defines` you have to think other things, like if you need to recreate all expressions.